src/components/InputTodo.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+    };
+  }
```

Comment:
This would be a nightmare to refactor into function-based, wouldn't it? hahah

Reply:
It could be, but since it only holds one state, maybe not as hard as the TodoContainer.js